versions/run.ver_hercules

```diff
@@ -8,8 +8,8 @@ export modulepath_spack_stack_unienv="/work/noaa/epic/role-epic/spack-stack/herc
 export crtm_ver="2.4.0.1"
 export esmf_ver="8.6.0"
 export fms_ver="2023.04"
```

Comment:
Is there a reason why `fms_ver` wasn't updated to `2024.01` for `run.ver_hercules`?

Reply:
Good point!!! :) I'll update it.

Reply:
Thanks, @chan-hoo!